src/components/PassphraseInput.js

```diff
@@ -114,7 +116,7 @@ class PassphraseInput extends Nimiq.Observable {
 }
 
 PassphraseInput.Events = {
-    PASSPHRASE_ENTERED: 'passphrase-entered'
+    PASSPHRASE_ENTERED: 'passphrase-entered',
```

Comment:
I guess I don't like this rule that much

Reply:
Agreed that trailing commas are more practical.